charts/pocket/values.yaml

```diff
@@ -170,6 +170,8 @@ service:
   annotations: {}
```

Comment:
Was going to ask you some stuff, but learnt everything from ChatGPT.

![Screenshot 2023-05-22 at 4 07 50 PM](https://github.com/pokt-network/pocket/assets/1892194/531f40f7-dfc5-46ef-9223-86802bc8f8a8)